README.md

```diff
@@ -39,9 +39,11 @@ Read ["Installing Rust"] from [The Book].
     $ make && make install
     ```
 
-    > ***Note:*** You may need to use `sudo make install` if you do not
-    > normally have permission to modify the destination directory. The
-    > install locations can be adjusted by passing a `--prefix` argument
+    > ***Note:*** Currently `sudo make install` is not supported, so if you
+    > do not have permission to modify the destination directory then you'll
+    > want to install to a local directory like `$HOME/rust`.
```

Comment:
Why isn't `sudo make install` supported? That is _the_ command one typically needs to install Rust from source.